pyaedt/edb_core/edb_data/simulation_configuration.py

```diff
@@ -146,9 +188,28 @@ def airbox_negative_vertical_extent(self, value):  # pragma: no cover
         if isinstance(value, (int, float)):
             self._airbox_negative_vertical_extent = value
 
+    @property
+    def use_airbox_negative_vertical_extent_multiple(self):
+        """Negative extent multiple value of the airbox.
```

Comment:
```suggestion
        """Multiple for the negative extent of the airbox.
```
We should always try to avoid too many adjectives before a noun to increase comprehension.